### ISSUE TITLE
[Accessibility] Insufficient color contrast

### ISSUE THREAD

**Issue by gabrielwalt** [adobe/aem-core-wcm-components#1475]
Following components have insufficient color contrast on the [AEM Components Library](https://www.aemcomponents.dev/):
* Progress Bar
* Carousel

This can be analyzed with https://www.tpgi.com/color-contrast-checker/
